test/cppsuite/tests/hs_cleanup.cpp

```diff
@@ -111,7 +103,6 @@ class hs_cleanup : public test {
             testutil_assert(rollback_retries < MAX_ROLLBACKS);
         }
         /* Ensure our last transaction is resolved. */
-        if (tc->txn.active())
```

Comment:
nitpick: I think the original implementation is easier to understand here.

Reply:
As discussed, I kept the changes.